static/teams/bots-graphs.html

```diff
@@ -1,173 +0,0 @@
-  });
-
-  var chart = new Keen.Dataviz()
-    .el("#keen-example-chart")
-    .height(320)
-    .title("Activity / repository / week on a 52 week period")
-    .type("bar")
-    .stacked(true)
-    .prepare();
-
-  client
-    .query("count", {
-      event_collection: "dispatch",
-      filters: [
-        {
-          operator: "not_contains",
-          property_name: "mention.repository",
-          property_value: "/test1",
-        },
-      ],
-      group_by: ["mention.repository"],
-      interval: "weekly",
-      timeframe: "this_52_weeks",
-      timezone: "US/Pacific",
-    })
-    .then(function (res) {
-      chart.data(res).render();
-    })
-    .catch(function (err) {
-      chart.message(err.message);
-    });
-
-  var chart2 = new Keen.Dataviz()
-    .el("#keen-chart-2")
-    .height(340)
-    .width("400px")
-    .title("Number of backport/Year/Project.")
-    .type("table")
-    .prepare();
-
-  client
-    .query("count", {
-      event_collection: "dispatch",
-      filters: [
-        {
-          operator: "contains",
-          property_name: "mention.command",
-          property_value: "backport",
-        },
-      ],
-      group_by: ["mention.organisation"],
-      timeframe: "this_5_years",
-      timezone: "US/Pacific",
-    })
-    .then(function (res) {
-      var data = chart2.data(res);
-      console.log(data);
-      data.dataset = data.dataset.sortRows("desc", function (arr) {
-        console.log(arr);
-        return arr[1];
-      });
-      data.dataset.matrix[0] = ["Organisation", "Backports"];
-      data.render();
-    })
-    .catch(function (err) {
-      chart2.message(err.message);
-    });
-
-  var chart3 = new Keen.Dataviz()
-    .el("#total-pr")
-    .height(340)
-    .title("All attempted backports (excluding test repositories)")
-    .type("metric")
-    .prepare();
-
-  client
-    .query("count", {
-      event_collection: "dispatch",
-      filters: [
-        {
-          operator: "contains",
-          property_name: "mention.command",
-          property_value: "backport",
-        },
-        {
-          operator: "not_contains",
-          property_name: "mention.repository",
-          property_value: "test1",
-        },
-      ],
-      timeframe: "this_14_years",
-      timezone: "US/Pacific",
-    })
-    .then(function (res) {
-      chart3.data(res).render();
-    })
-    .catch(function (err) {
-      chart3.message(err.message);
-    });
-
-  var chart4 = new Keen.Dataviz()
-    .el("#backport-success")
-    .height(240)
-    .title("Backport Success")
-    .stacked(true)
-    .type("columnchart")
-    .prepare();
-
-  client
-    .query("count", {
-      event_collection: "backport_stats",
-      filters: [
-        {
-          operator: "not_contains",
-          property_name: "slug",
-          property_value: "test1",
-        },
-        { operator: "ne", property_name: "reason", property_value: "Sucess" },
```

Comment:
`/Sucess/Sucess/`

Reply:
This is code, I would add the file to the exceptions as the value might be typoed upstream if something is looking for this property value.